### PR TITLE
Code Quality: Refactor StatusCenter-related codes

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Motivation & Context

- Improved readability

> **Warning**<br/>Please do not review code lines that I did not changed. I cannot make more diff. If there're more codes that are preferred to change, I will do that and mark that PR as the final one for that stage.

### PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #4180
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

image

### CODE REVIEW

File: src/Files.App/ViewModels/UserControls/StatusCenterViewModel.cs [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 namespace Files.App.ViewModels.UserControls
 {
-	public class OngoingTasksViewModel : ObservableObject, IOngoingTasksActions
+	public class StatusCenterViewModel : ObservableObject
 	{
-		public ObservableCollection<StatusBanner> StatusBannersSource { get; private set; } = new ObservableCollection<StatusBanner>();
-
-		private int medianOperationProgressValue = 0;
+		private int _MedianOperationProgressValue = 0;
 		public int MedianOperationProgressValue
```
Comment (ferrariofilippo) [inline src/Files.App/ViewModels/UserControls/StatusCenterViewModel.cs:11]: Can you rename it to `AverageOperationProgressValue`? 
In `UpdateMedianProgress()` (which should be renamed as well) we are calculating the average value not the median

File: src/Files.App/Utils/StatusCenter/StatusCenterItem.cs [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+	public class StatusCenterItem : ObservableObject
+	{
+		private readonly float _initialProgress = 0.0f;
+
+		private string? _FullTitle;
+		public string? FullTitle
+		{
+			get => _FullTitle;
+			set => SetProperty(ref _FullTitle, value ?? string.Empty);
+		}
```
Comment (ferrariofilippo) [inline src/Files.App/Utils/StatusCenter/StatusCenterItem.cs:21]: I don't get why you are assigning `string.Empty` when `value is null` even if you marked the property as nullable. I think you can initialize it to `string.Empty` and make it not nullable

**Review by ferrariofilippo — COMMENTED** [review]
It works but some things might be refined

File: src/Files.App/Utils/StatusCenter/StatusCenterItem.cs (hunk 2 above)
Comment (0x5bfa) [reply src/Files.App/Utils/StatusCenter/StatusCenterItem.cs:21]: Actually this is what this was. I will change it.

**Pull request closed without merging**